### ISSUE TITLE
Plugin to get many things on the status bar.

### ISSUE THREAD

**Issue by viaahmed** [chetachiezikeuzor/MySnippets-Plugin#3]
👋Hi @chetachiezikeuzor 
Thanks for your beautiful and awesome plugin. It's a life saver.♥️💊

I was waiting for such solution since months. The way things get compact in the status bar popup, really make it easy to navigate, use and relish the cuteness of the UI.

I was suggesting there may be a plugin which can bring almost everything thing on status bar

like community plugin settings as requested in #1

Other options like
- local graph
- global search
- file explorer 🔺
- tag pane
- currently open files (useful in full focus mode) (low priority feature)
- outline
- calender
- any note/leaf/multimedia ( ie, a note resting in the popup just like how a note can stay in a sidebar )

So, this way we can tuck in icons all along the status bar and **forget** the sidebars.

It would be added benefit if the lined up icons could be fired using hotkeys (preferably function keys, especially single keys without modifier)

Summing up, making this great plugin would like crossing the cMenu plugin and MySnippets plugin and the status bar.

Thanks again 🤗
